### PR TITLE
Fix Xcode 12 warnings

### PR DESCRIPTION
**Pull request by hewigovens**
What changed:

1. Fix quoted include warnings
2. explicitly include `<cassert>`
3. remove hardcoded iOS 13.3

### CODE REVIEW

File: include/TrustWalletCore/TWAnySigner.h [hunk 1]
```diff
@@ -5,10 +5,10 @@
 // file LICENSE at the root of the source code distribution tree.
 #pragma once
 
-#include "TWBase.h"
-#include "TWCoinType.h"
-#include "TWData.h"
-#include "TWString.h"
+#include <TrustWalletCore/TWBase.h>
+#include <TrustWalletCore/TWCoinType.h>
```
Comment (vikmeup) [inline include/TrustWalletCore/TWAnySigner.h:9]: Can we rename TrustWalletCore to WalletCore?

**Comment by catenocrypt** [comment]
Include cleanup is a welcome change!

File: include/TrustWalletCore/TWAnySigner.h (hunk 1 above)
Comment (hewigovens) [reply include/TrustWalletCore/TWAnySigner.h:9]: just include folder?

Comment (catenocrypt) [reply include/TrustWalletCore/TWAnySigner.h:9]: Include is too general, if other CPP project uses it, it is better to be the project name.  WalletCore is OK, but would this not be a breaking change?  Or not too many CPP projects using wallet core out there?

Comment (vikmeup) [reply include/TrustWalletCore/TWAnySigner.h:9]: Is include only headers files? Maybe rename to headers

Comment (vikmeup) [reply include/TrustWalletCore/TWAnySigner.h:9]: This is definitely for another PR, but something to improve in naming

Comment (hewigovens) [reply include/TrustWalletCore/TWAnySigner.h:9]: `TrustWalletCore` is also the iOS framework name, just Headers don't make any sense

**Pull request merged**